config.txt

```diff
@@ -0,0 +1,49 @@
+{
+    "LATMAX": "19.0", 
```

Comment:
@youssefbiaz I would re-organize this sample config file a bit and group variables together.  JSON doesn't support comments, so a logical grouping would help users navigate the config easier.  If we really want to include comments than a different file format like a config.ini or yaml should be used.  I think we can keep the json format for now.